docs/src/pages/components/tabs/AccessibleTabs.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+interface DemoTabsProps {
+  labelId: string;
+  onChange: (event: unknown, value: number) => void;
+  selectionFollowsFocus?: boolean;
+  value: number;
+}
+function DemoTabs(props: DemoTabsProps) {
+  const { labelId, onChange, selectionFollowsFocus, value } = props;
+
+  return (
+    <AppBar position="static">
+      <Tabs
+        aria-labelledby={labelId}
+        onChange={onChange}
+        selectionFollowsFocus={selectionFollowsFocus}
+        value={value}
+      >
+        <Tab label="Item One" aria-controls="simple-tabpanel-0" id="simple-tab-0" />
```

Comment:
It seems to duplicate with another id on the page https://validator.w3.org/nu/?doc=https%3A%2F%2Fmaster--material-ui.netlify.app%2Fcomponents%2Ftabs%2F.